test/uk/gov/hmrc/icedsubscriptionfrontend/views/AlreadyEnrolledPageSpec.scala

```diff
@@ -57,24 +59,41 @@ class AlreadyEnrolledPageSpec extends SpecBase {
     }
 
     "have a only one page heading" in {
-      document.select(Selectors.h1).text shouldBe "Enrol with the Safety and Security service"
+      document.select(Selectors.h1).text shouldBe "Your organisation is already enrolled with S&S GB"
       document.select(Selectors.h1).size shouldBe 1
     }
 
-    "have inset text for with notice" in {
+    "have indication of when active" in {
       content
         .select(Selectors.paragraph)
         .first
-        .text shouldBe "The GB EORI you supplied is already enrolled with the Safety and Security service."
+        .text shouldBe "The GB EORI you supplied (GB1234567890) is already enrolled with S&S GB."
```

Comment:
Can we have a test where the eori is not defined (so as to ensure that we don't end up with:

> The GB EORI you supplied () is ...